pyuvdata/uvdata/corr_fits_src/corr_fits.pyx

```diff
@@ -37,15 +37,18 @@ cpdef dict input_output_mapping():
 
   return pfb_inputs_to_outputs
 
-cpdef tuple generate_map(
+@cython.boundscheck(False)
+@cython.wraparound(False)
+cpdef void generate_map(
   dict ants_to_pf,
-  dict in_to_out,
-  numpy.ndarray[ndim=1, dtype=numpy.int32_t] map_inds,
-  numpy.ndarray[ndim=1, dtype=numpy.npy_bool] conj
+  numpy.int32_t[:] map_inds,
```

Comment:
Since you're defining these in python using `np.zeros` (and are thus contiguous), is there a benefit to declaring these memory views as, e.g., `numpy.int32_t[::1]`? (Genuinely curious, I haven't done the benchmarking myself.)

Reply:
Do you mean the `map_inds` being a memoryview now? The overhead is (allegedly) _slightly_ lower for memoryviews, and allows for direct C access to the arrays. This way we don't have to bloat up the function with additional declarations of memoryviews. I have found in some tests in other places that memoryview inputs are a little faster. But I think my preference is still in the anecdotal category. The cython docs do emphasize that the memoryviews are "newer and better" and more flexible over all.

Reply:
I meant more that we use the `::1` indexing syntax, rather than just `:`. I believe the former further requires the memory to be C-contiguous (if I'm interpreting the docs correctly), but like the initial declaration discussed in the python code below, I don't think it really matters for a 1-d array. Let's leave it as is.

Reply:
Oh yeah that's a good point. especially since I'm just using `[:]`. it doesn't matter. We should just pick one, would you prefer `[::1]` and leave the python ordering or remove the python ordering?

Reply:
Let's index it using `::1` here (maybe with a comment explaining that it requires a C-contiguous array), and keep the `order` argument in python-land. Probably better to be explicit and model best practices, even if it is a moot point for 1d array.